Classes/Domain/Model/Changes/AbstractMove.php

```diff
@@ -40,7 +41,11 @@ protected function finish(NodeInterface $node)
             $updateNodePath->setNewContextPath($this->getSubject()->getContextPath());
             $this->feedbackCollection->add($updateNodePath);
         }
-
+        if ($this->getSubject()->getNodeType()->isOfType('Neos.Neos:Document')) {
```

Comment:
this seems to work well. and for content nodes it seems that `RenderContentOutOfBand` takes care of highlighting the node where it was moved to.

Reply:
yes, I was pretty happy to find something that works out of the box since I did not want to trigger addNodeCreatedFeedback on a node move